dockerfiles/rpm.dockerfile

```diff
@@ -116,7 +117,8 @@ RUN createrepo /build \
  && rm -rf /build/repodata
 RUN containerd --version
 RUN ctr --version
-RUN runc --version
+# Don't try to run runc on RHEL/CentOS as we don't package it anymore.
+RUN if [ "${BASE}" != "rhel" ] && [ "${BASE}" != "centos" ]; then runc --version; fi
```

Comment:
I'd rather keep this change for a separate PR (sorry haven't been able to spend time to respond on the linked ticket); this definitely needs a wider discussion (and not something we should change overnight).

I'm ok with adding the `stream9` to the list in CI though (note that this Jenkinsfile is used for CI, but the actual matrix for which we built is in our internal release pipeline, so it may not be included (yet) on the next build)

Reply:
I have updated this pull request to focus just on CentOS 9. I have split it in two commits, the second commit being now tracked in #272.